File: src/items/weapons/revolver.py
from gun import Gun
from revolver_bullet import RevolverBullet

class Revolver(Gun):
  def __init__(self, init_rounds=6):
    self.rounds = init_rounds
    super(Revolver, self).__init__(RevolverBullet, clip_size=6, bullet_range=10)

File: src/server/items/weapons/revolver.js
import Gun from './gun';
import RevolverBullet from './revolverBullet';

export default class Revolver extends Gun {
  constructor(kwargs = {}) {
    super(kwargs);
    this.currentClipAmount = 6;
    this.totalClipAmount = 6;
    this.bulletDistance = 1;
    this.bulletConstructor = RevolverBullet;
  }
}
